src/uu/wc/src/wc.rs

```diff
@@ -538,9 +553,9 @@ enum CountResult {
     /// Nothing went wrong.
     Success(WordCount),
     /// Managed to open but failed to read.
-    Interrupted(WordCount, io::Error),
+    Interrupted(WordCount, UIoError),
     /// Didn't even manage to open.
-    Failure(io::Error),
+    Failure(UIoError),
```

Comment:
This seems weird, because then we have the context of the `UIoError` wrapped in the context provided by the enum? Conversion from `io::Error` to `UError` to `WcError` to `UError` seems like a weird way to do things.